hooks/tk-vred_scene_operations.py

```diff
@@ -143,7 +147,8 @@ def update_items(self, items):
 
         # Update the VRED references based on their reference type being source or smart
         if refs_to_load:
-            self._vredpy.vrReferenceService.loadSourceReferences(refs_to_load)
+            interactive_update = self.parent.get_setting("interactive_update", False)
```

Comment:
Question: why do we always give the setting's default value in the second argument of this function while the setting has a default value and this function is supposed to return it if it's not set.
It looks like we duplicate the default value everywhere for no reason.

Reply:
@julien-lang the default value given to `get_setting` will only be used if the setting itself does not define a default value. It is confusingly a "default value for the default value". Since our setting does define a default value, we could omit this but it is added safety, and mostly just out of habit that I defined a fallback default value here.